src/elexmodel/client.py

```diff
@@ -86,6 +86,15 @@ def _check_input_parameters(
             raise ValueError("handle_unreporting must be either `drop` or `zero`")
         return True
 
+    def get_conformalization_data_unit(self):
+        return self.conformalization_data_unit_dict
+
+    def get_conformalization_data_agg(self):
+        return self.conformalization_data_agg_dict
+
+    # TODO comment: conformalization data pulled out from get_estimates function below
```

Comment:
to clarify this. can you add a comment to the above functions saying what they return and specifically that they return `None` if `get_estimates` hasn't been called yet.

Reply:
done